heartbeat/mysql

```diff
@@ -544,14 +568,20 @@ set_master() {
     # Informs the MySQL server of the master to replicate
     # from. Accepts one mandatory argument which must contain the host
     # name of the new master host. The master must either be unchanged
-    # from the laste master the slave replicated from, or freshly
+    # from the last master the slave replicated from, or freshly
     # reset with RESET MASTER.
+    if [ -n "$OCF_RESKEY_replication_master_ssl_ca" -a -n "$OCF_RESKEY_replication_master_ssl_cert" -a -n "$OCF_RESKEY_replication_master_ssl_key" ]; then
+        master_ssl_params=", MASTER_SSL=1, \
```

Comment:
This parameter should be defined as an empty parameter before the `if` to avoid any issues undefined variables might cause.

Reply:
Correct, will be updated.